sdk/sdktypes/symbol.go

```diff
@@ -18,7 +18,7 @@ var symbolRE = kittehs.Must1(regexp.Compile(`^[a-zA-Z_][\w]*$`))
 
 func (symbolTraits) Validate(s string) error {
 	if s != "" && !symbolRE.MatchString(s) {
-		return errors.New("invalid")
+		return errors.New("illegal symbol (expected [a-zA-Z_][\\w]*)")
```

Comment:
not sure putting the regex here is a good idea :)

Reply:
@itayd @dabbr please suggest the way you want it. I could either remove it as @itayd  suggested or use a constant string as @dabbr suggested. I think it's important to provide user with enough and clear info.